tools/onni/CountOperatorsPass.cpp

```diff
@@ -27,6 +30,9 @@ Pass::ReturnType CountOperatorsPass::runOnModule(Module &pModule)
   uint64_t total = 0;
 
   for (ComputeOperator &cm : *pModule.getRootComputeGraph()) {
+    if (dyn_cast<InputOperator>(&cm)) continue;
```

Comment:
Replace with a function? IsIOWOperator() or SkipInCountStatistics()?

Reply:
Ok, that's Great! Thanks a lot!